src/ResourceManager/Sql/Commands.Sql/Failover Group/Cmdlet/NewAzureSqlFailoverGroup.cs

```diff
@@ -63,8 +65,10 @@ public class NewAzureSqlFailoverGroup : AzureSqlFailoverGroupCmdletBase
         /// <summary>
         /// Gets or sets the grace period with data loss for the Sql Azure Failover Group.
         /// </summary>
+        [Alias("GracePeriodWithDataLossHours")]
```

Comment:
@ziwa-msft not needed at this stage, introduce this when actually changing the parameter name.